scripts.py

```diff
@@ -376,3 +394,27 @@ def run(self, uploader=None, cmd_args=None):
             with open(filename, 'w') as f:
                 f.write(unicode(feed))
             self.log.info("OPDS feed saved locally at %s", filename)
+
+        if parsed.search_url and parsed.search_index:
+            search_client = ExternalSearchIndex(parsed.search_url, parsed.search_index)
+            annotator = ContentServerAnnotator()
+
+            search_documents = []
+
+            # It's slow to do these individually, but this won't run very often.
+            for work in works:
+                doc = work.to_search_document()
+                doc["_index"] = search_client.works_index
+                doc["_type"] = search_client.work_document_type
+                doc["opds_entry"] = etree.tostring(AcquisitionFeed.single_entry(self._db, work, annotator))
```

Comment:
I couldn't create a batch of search documents in a database query like I do in the circulation manager search index script, because I needed to add in the annotated OPDS entry. I considered using one of the OPDS entries in the db, but it doesn't have the open access link. 

As a result, this duplicates some code from ExternalSearchIndex.bulk_update.